xonsh/ptk/completer.py

```diff
@@ -41,7 +41,11 @@ def get_completions(self, document, complete_event):
 
     def reserve_space(self):
         cli = builtins.__xonsh_shell__.shell.prompter.cli
-        window = cli.application.layout.children[1].children[1].content
+        #TODO remove after next prompt_toolkit release
+        try:
+            window = cli.application.layout.children[1].children[1].content
+        except AttributeError:
+            window = cli.application.layout.children[1].content
```

Comment:
heh.  oops.  we want to keep the second one.  I'll put in a comment to that effect.